manifests/i/Intel/ProcessorDiagnosticTool/4.1.5/Intel.ProcessorDiagnosticTool.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Intel.ProcessorDiagnosticTool
+PackageVersion: 4.1.5
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+  InstallerUrl: https://downloadmirror.intel.com/19792/eng/ipdt_installer_4.1.5.37_64bit.msi
+  InstallerSha256: FAFE723C3FF07086183693BEE910F8240AED9B1B6698008C0410DBC2F95CCAE4
+  ProductCode: '{6E05E656-6ED8-49DE-AA9C-C4677F7086C5}'
```

```suggestion
  ProductCode: '{6E05E656-6ED8-49DE-AA9C-C4677F7086C5}'
- InstallerLocale: en-US
  Architecture: x86
  InstallerType: msi
  InstallerUrl: https://downloadmirror.intel.com/19792/eng/ipdt_installer_4.1.5.37_32bit.msi
  InstallerSha256: 05168596E7B5F7F5778125858454C0E4BB7A04997F67274831C7F5F24D058C6C
  ProductCode: '{57DD224D-404E-49FE-AB9A-A77A7C3CB02D}'
```

---

Isn't 64 bit better than 32 bit?

---

> ```suggestion
> 
>   ProductCode: '{6E05E656-6ED8-49DE-AA9C-C4677F7086C5}'
> 
> - InstallerLocale: en-US
> 
>   Architecture: x86
> 
>   InstallerType: msi
> 
>   InstallerUrl: https://downloadmirror.intel.com/19792/eng/ipdt_installer_4.1.5.37_32bit.msi
> 
>   InstallerSha256: 05168596E7B5F7F5778125858454C0E4BB7A04997F67274831C7F5F24D058C6C
> 
>   ProductCode: '{57DD224D-404E-49FE-AB9A-A77A7C3CB02D}'
> 
> ```

Isn't 64 bit better than 32 bit?

---

This suggestion adds the x32 for compatability